src/Tickets/Seating/Assets.php

```diff
@@ -57,13 +56,14 @@ public function get_utils_data(): array {
 				'layout-edit' => Layout_Edit::get_edit_url_by_post( get_the_ID() ),
 			],
 			'localizedStrings' => [
-				'capacity-form' => $localization->get_capacity_form_strings(),
-				'dashboard'     => [ 'seats-action-label' => _x( 'Seats', 'Ticket Dashboard actions', 'event-tickets' ) ],
-				'maps'          => [
+				'capacity-form'  => $localization->get_capacity_form_strings(),
+				'capacity-table' => [ 'seats-row-label' => _x( 'Assigned Seating', 'Capacity table row label for assigned seating tickets', 'event-tickets' ) ],
```

Comment:
This should come up in a snapshot; if not already covered by tests, cover it now.